lti_consumer/lti_1p1/oauth.py

```diff
@@ -147,7 +147,11 @@ def log_authorization_header(request, client_key, client_secret):
     oauth_body_hash = str(base64.b64encode(sha1.digest()))
     log.debug("[LTI] oauth_body_hash = %s", oauth_body_hash)
     client = oauth1.Client(client_key, client_secret)
-    params = client.get_oauth_params(request)
+    params = oauth1.rfc5849.signature.collect_parameters(
+        body=request.body,
+        headers=request.headers,
+        exclude_oauth_signature=False
+    )
```

Comment:
@Agrendalath Debugging the Oauth failure, I looked at the source for [Webob request](https://github.com/Pylons/webob/blob/259230aa2b8b9cf675c996e157c5cf021c256059/src/webob/request.py#L941) body attribute and the logic in the Oauthlib's `get_oauth_params` [function](https://github.com/oauthlib/oauthlib/blob/master/oauthlib/oauth1/rfc5849/__init__.py#L222). While Webob maintains the content of a request body as binary, oauthlib expects it to be a string.

Both this line here and the oauthlib's code are at least 7 years old and I am not sure how this mismatch didn't raise an issue before. Maybe because, this function was always called with `www-form-urlencoded` data?? 

Anyway, I found that there is alternate function which accpets binary body and can basically do the exact same function in [Line 92](https://github.com/openedx/xblock-lti-consumer/blob/master/lti_consumer/lti_1p1/oauth.py#L92). I have reused it here as an replacement and it seems to work without any issues.

Reply:
Ah.. the CI has caught an error. debugging again